### ISSUE TITLE
On regularization of gradient descent, layer imbalance and flat minima

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#69]
NeurIPS2020 OPT2020の論文
LARSの人

https://opt-ml.org/posters/2020/poster_12.png
https://arxiv.org/abs/2007.09286

> We analyze the training dynamics for deep linear networks using a new metric - layer imbalance - which defines the flatness of a solution. We demonstrate that different regularization methods, such as weight decay or noise data augmentation, behave in a similar way. Training has two distinct phases: 1) optimization and 2) regularization. First, during the optimization phase, the loss function monotonically decreases, and the trajectory goes toward a minima manifold. Then, during the regularization phase, the layer imbalance decreases, and the trajectory goes along the minima manifold toward a flat area. Finally, we extend the analysis for stochastic gradient descent and show that SGD works similarly to noise regularization.

<img width="1792" alt="Screen Shot 2020-12-06 at 18 13 46" src="https://user-images.githubusercontent.com/8721858/101276245-cf06b700-37ee-11eb-9d7a-b6a4707f3006.png">
